.github/workflows/release.yml

```diff
@@ -3,7 +3,6 @@ name: Release
 on:
   push:
     branches:
-      - develop
```

Comment:
After this, we will only publish real versions once we merge to `master` like [this](https://github.com/ethereum-optimism/optimism/pull/947)